### ISSUE TITLE
Write results to cope with PyPsa

### ISSUE THREAD

**Issue by arjunane** [wupperinst/osmTGmod#18]
Results shall optionally (and maybe also by default) be formated in a way, that PyPsa can red them.

**Comment by maltesc** [comment]
This will be done by ZNES, too. Let's create synergy here!

**Comment by arjunane** [comment]
Good point. Let us just make the output choosable. :)

**Comment by maltesc** [comment]
Yes, this will happen. The idea is to simply add another "export_csv" function, that automatically calculates the values and exports the corresponding tables.
But this is only one part of it, since you will need the new structure also within the database (in order to be able to dynamically access the different results), we need to change the result-database structure. What is your idea?

**Comment by maltesc** [comment]
We decided to create an independent tool that reads the osmTGmod result data, processes it in Python and writes PyPSA tables into another database (which in ZNES's case is the project database, but this can be easily adapted). This tool will be available soon.

**Comment by MatthiasKl** [comment]
Is this tool already available? I am about to write something to convert osmtgmod csv output data to pypsa csv input data - but would of course prefer to use something existing rather than to code on my own.

**Comment by arjunane** [comment]
Hi MatthiasKI, 
this tool is unfortunatelly not available. As osmTGmod already populates a PostgresDB, i just read the information i need with psycopg2 and SQL-Querrys into python and populate the PyPSA network with it. Takes some time but works quite smoothly. If you come up with a osmTGmod inbuild solution, i will be more than happy to integrate that into the code. Just start a pull-request.

**Comment by MatthiasKl** [comment]
Thank you for the advice. I did it like this, only started on the csv output instead of the PostgresDB. It tunred out to be only few lines of coding, the main "challenge" was transformation of coordinate notation from wkb to normal numbers. Furtunately, shapely does this. So after all, indeed no need for whatever tool to get the network to pypsa.
